pages/linux/disown.md

```diff
@@ -14,3 +14,7 @@
 - Disown all jobs:
 
 `disown -a`
+
+- Disown job from current shell (SIGHUP not sent to job on shell exit):
```

Comment:
How is this different from a regular disown? I'm not sure I understand.

Reply:
Sorry, my wording was completely off (long Friday), 
the -h flag will not remove the job from the jobs table, but will rather keep the job in the job table but mark it so it does not receive any future SIGHUP signal from a shell exit. 
Wording has been changed now, hope its better, thanks

Reply:
from bash man page/built in commands/
 If the -h option is given, each jobspec is not removed from the table, but is marked so that SIGHUP is not sent to the job if the shell receives a SIGHUP

Reply:
Ah, I see! No problem. The new wording looks much better :-)

Useful addition!